src/gui/components/navbar/navbar.jsx

```diff
@@ -1,17 +1,23 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { merge } from 'glamor'
+
+import { primaryColor } from '../../styles'
+import styles from './navbar.styles'
 
 const Navbar = ({ style, className }) => {
   return (
-    <div style={style} className={className}>
-      Workbench
+    <div style={style} className={merge([styles.layout, primaryColor, className])}>
```

Comment:
- Don't use `merge([])` but `merge(a, b, c)`
 - Move `primaryColor` to the style file